src/Protocol/Protocol17x.cpp

```diff
@@ -3280,9 +3280,9 @@ void cProtocol176::SendPlayerSpawn(const cPlayer & a_Player)
 
 	for (Json::Value::iterator itr = Properties.begin(), end = Properties.end(); itr != end; ++itr)
 	{
-		Pkt.WriteString(((Json::Value)*itr).get("name", "").asString());
-		Pkt.WriteString(((Json::Value)*itr).get("value", "").asString());
-		Pkt.WriteString(((Json::Value)*itr).get("signature", "").asString());
+		Pkt.WriteString(static_cast<Json::Value>(*itr).get("name", "").asString());
+		Pkt.WriteString(static_cast<Json::Value>(*itr).get("value", "").asString());
+		Pkt.WriteString(static_cast<Json::Value>(*itr).get("signature", "").asString());
```

Comment:
Doesn't the iterator dereference into the `Json::Value` itself, does it really require the cast? Shouldn't a simple `itr->get(...` work?